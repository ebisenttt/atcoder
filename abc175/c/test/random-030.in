-421228639287949 366835130366745 712359462232449
