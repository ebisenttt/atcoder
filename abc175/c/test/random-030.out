291130822944500
